test/encoding.ts

```diff
@@ -0,0 +1,104 @@
+import { BigNumber } from "ethers";
+import { ethers } from "hardhat";
+
+import { Order } from "../src/ts";
+
+export type RawOrder = [
```

Comment:
What's the difference between Order and RawOrder? Maybe just a comment and what you consider as Raw would help.

<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia3.giphy.com%2Fmedia%2FMnpPCugwALAHsTygpd%2Fgiphy-downsized-medium.gif&token=raw" data-gitmeme-token="raw"><img src="https://media3.giphy.com/media/MnpPCugwALAHsTygpd/giphy-downsized-medium.gif" title="Created by gitme.me with /raw"/></a>

Reply:
Its the data that `ethers` returns from contract interactions. Maybe `AbiOrder`?

Reply:
Yes that would sound more descriptive to me.